web/src/components/TimeAverageToggle.tsx

```diff
@@ -28,12 +28,18 @@ export interface TimeAverageToggleProps {
 function TimeAverageToggle({ timeAverage, onToggleGroupClick }: TimeAverageToggleProps) {
   const { t } = useTranslation();
   const historicalLinkingEnabled = useFeatureFlag('historical-linking');
+  const is72HourEnabled = useFeatureFlag('72-hours');
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { HOURLY_72, ...standardTimeAverages } = TimeAverages;
+  const timeOptions = is72HourEnabled ? TimeAverages : standardTimeAverages;
```

Comment:
```suggestion
  const timeOptions = is72HourEnabled 
    ? Object.values(TimeAverages)
    : Object.values(TimeAverages).filter(option => option !== TimeAverages.HOURLY_72);
```

This could help not having an unused var